tutorial/exercise_03/instructions.md

```diff
@@ -36,21 +36,21 @@ More information on this topic can be found in [here](http://vunit.github.io/pyt
     set VUNIT_SIMULATOR=<simulator>
 	```
 
-* Open `exercise_03/run.py` and notice how the `lib` library is added
+* Open `exercise_03/run.py` and notice how the [external library](http://vunit.github.io/py/vunit.html#vunit.ui.VUnit.add_external_library) `lib` is added
 
   ``` python
   lib = prj.add_external_library("lib", join(root, "..", "exercise_01", "vunit_out", "<your simulator>", "libraries", "lib"))
   ```
 
   Sometimes it's useful to add libraries that have been compiled external to VUnit, for example vendor libraries and third-party IPs. In this case we're just taking what's already compiled in exercise 01.
 
-* Run all tests with the verbose flag
+* Run all tests with the verbose flag (make sure that you have run exercise01 first, so lib library from exercise01 is compiled)
 
     ``` console
     python run.py -v
     ```
 
-    Note the printed counter width which is the default value.
+    Note the printed counter width for all tests is the default value.
```

Comment:
I think the original sentence was missing a comma between `width` and `which`. In this new wording, I'm missing a `that` between `Note` and `the`.